tools/test-js-with-node

```diff
@@ -106,6 +106,7 @@ EXEMPT_FILES = {
     "static/js/poll_widget.js",
     "static/js/popovers.js",
     "static/js/popover_menus.js",
+    "static/js/read_receipts.js",
```

Comment:
I'll try to add tests and remove this. Any ideas on how to test the logic in `read_receipts.js` would be very helpful.